core-services/prow/02_config/openshift/velero-plugin-for-aws/_prowconfig.yaml

```diff
@@ -3,28 +3,12 @@ branch-protection:
     openshift:
       repos:
         velero-plugin-for-aws:
-          branches:
```

Comment:
change here applies `allow_force_pushes` to all branches, but I think is fine, we own that repo